pkgs/tools/misc/twm/default.nix

```
@@ -5,24 +5,27 @@
 , openssl
 , pkg-config
 , Security
+, nix-update-script
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "twm";
-  version = "0.9.0";
+  version = "0.9.1";
 
   src = fetchFromGitHub {
     owner = "vinnymeller";
     repo = pname;
```

There is a preference to avoid such use of `pname` (see #277994)
```suggestion
    repo = "twm";
```